src/app/src/components/annotations/annotator.tsx

```diff
@@ -652,22 +662,43 @@ export default class Annotator extends Component<
     this.filterAnnotationVisibility();
   };
 
-  private updateImage = () => {
+  private updateCacheList = async () => {
+    if (this.props.loadedModel) {
+      await APIGetCacheList(this.props.loadedModel.hash)
+        .then(res => {
+          this.setState({ cacheList: res.data });
+        })
+        .catch(error => {
+          let message = `Failed to load cached predictions. ${error}`;
+          if (error.response) {
+            message = `${error.response.data.error}: ${error.response.data.message}`;
+          }
+          CreateGenericToast(message, Intent.DANGER, 3000);
```

Comment:
@beesaycheese I prefer if this fails silently - there's nothing the user can do about this and since the cache is a "silent" QoL, just set all the `isCached` to false when this happens as you cannot be certain that there is a cache "hit" waiting for you.

Separately, the users don't know that theres a cache in place, so this is kind of error should be removed

Reply:
changes made, setting it as an empty list